core/GlobalState.cc

```diff
@@ -938,7 +964,7 @@ NameRef GlobalState::getNameUnique(UniqueNameKind uniqueNameKind, NameRef origin
         bucketId = (bucketId + probeCount) & mask;
         probeCount++;
     }
-    Exception::raise("should never happen");
+    return core::Names::empty();
```

Comment:
empty is emtry string https://github.com/sorbet/sorbet/blob/master/core/tools/generate_names.cc#L285
cound this use https://github.com/sorbet/sorbet/blob/master/core/NameRef.h#L82 instead? which is the name that `!nameRef.exists()`